webapp/channels/src/components/user_settings/general/user_settings_general.tsx

```diff
@@ -1329,6 +1406,28 @@ export class UserSettingsGeneralTab extends PureComponent<Props, State> {
             const active = this.props.activeSection === sectionName;
             let max = null;
 
+            const getDisplayValue = (attributeValue: string | string[]) => {
+                if (!attributeValue || (!Array.isArray(attributeValue) && !attributeValue.length)) {
+                    return '';
+                }
+
+                if (attribute.type === 'select' || attribute.type === 'multiselect') {
+                    const attribOptions: PropertyFieldOption[] = attribute.attrs!.options as PropertyFieldOption[];
```

Comment:
1/5 non-blocker
My PR updates the typing a bit to make `attrs` required and a bit more defined; perhaps the assertion and casting could be simplified a bit after that